### PR TITLE
Feature/xml parse statement

### PR DESCRIPTION
**Pull request by amanPras**
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test cobol XML parse statement as per [docs](https://www.ibm.com/docs/en/cobol-zos/6.2?topic=statements-xml-parse-statement)

## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: server/engine/src/main/resources/resourceBundles/messages_en.properties [hunk 1]
```diff
@@ -108,3 +108,10 @@ jsonGenProcess.identifier2.groupItemError=%s group item must have unique data-na
 jsonGenProcess.identifier3.dataType=%s must be an integer data item defined without the symbol P in its picture string.
 jsonGenProcess.identifier6.dataType=%s should be elementary item with PIC X clause.
 jsonGenProcess.condition.dataType=% must be a level-88 item directly subordinate to %s.
```
Comment (amanPras) [inline server/engine/src/main/resources/resourceBundles/messages_en.properties:110]: @zeibura Please help verify below messages

Comment (zeibura) [reply server/engine/src/main/resources/resourceBundles/messages_en.properties:110]: the second one should be
%s must be an elementary item with a PIC X clause.

**Review by zeibura — CHANGES_REQUESTED** [review]
see inline

File: server/engine/src/main/resources/resourceBundles/messages_en.properties (hunk 1 above)
Comment (amanPras) [reply server/engine/src/main/resources/resourceBundles/messages_en.properties:110]: updated

**Review by zeibura — APPROVED** [review]

**Review by ishche — APPROVED** [review]

**Pull request merged**